lib/Runtime/Language/ModuleNamespace.h

```diff
@@ -20,7 +20,11 @@ namespace Js
     public:
         static ModuleNamespace* GetModuleNamespace(ModuleRecordBase* moduleRecord);
 
-        static bool Is(Var ns) { return ModuleNamespace::Is(ns); }
+        static bool Is(Var ns)
+        {
+            // This function calls itself ?
+            return false; // ModuleNamespace::Is(ns);
```

Comment:
Would be good to add an assert(false) to know when this is called. Crawler will probably catch this.